app/helpers/application_helper.rb

```diff
@@ -6,4 +6,10 @@ def current_year
   def github_url(author, repo)
     link_to 'GitHub', "http://github.com/#{author}/#{repo}", target: '_blank'
   end
+
+  def flash_message type
```

Comment:
```suggestion
  def flash_message(type)
```
при вызове иногда пишут без скобок (обычно для встроенных методов только), но при определении метода точно нужны скобки